include/seastar/testing/perf_tests.hh

```diff
@@ -24,6 +24,7 @@
 #include <atomic>
 #include <memory>
 
+#define FMT_DEPRECATED_OSTREAM
```

Comment:
Please, explain this change in more details

Reply:
@xemul https://github.com/fmtlib/fmt/releases/tag/9.0.0
According to the release notes of fmt 9.0.0, automatic std::ostream insertion operator (operator<<) discovery has been disabled and it's also hard to define formatter specializations for the anonymous struct.

So it's just a workaround.

Reply:
It's better to fix it for real, since deprecated features eventually get removed.

Reply:
@avikivity Right, if we can move the struct duration to a named namespace, then we can define the formatter specialization.
Anyway, I'll take a try